exporter/opentelemetry-exporter-opencensus/src/opentelemetry/exporter/opencensus/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "0.18.dev0"
+__version__ = "0.17.b0"
```

Comment:
this will cause a failure during the publishing step of the build to pypi, i vote to release a new version that depends on the 1.0.0 release

Reply:
I changed `build.sh` to only build the packages we want and spit them into `dist/*`. Thus the build script in github actions will only pick up the ones we want.

Reply:
Plus, releasing a new version 0.18b won't work because it cannot depend on `1.0.0rc1` since `1.0.0rc1` wouldn't exist yet.